xbmc/interfaces/builtins/SkinBuiltins.cpp

```diff
@@ -492,6 +536,24 @@ static int SkinDebug(const std::vector<std::string>& params)
 ///     @param[in] url                   Extra URL to allow selection from (optional).
 ///   }
 ///   \table_row2_l{
+///     <b>`Skin.SetColor(string\,header[\,colorfile\,selectedcolor])`</b>
+///     ,
```

Comment:
```suggestion
///     \anchor Builtin_SetColor,
```

In order to reference a link to `Builtin_SetColor` below you'd need to define the `Builtin_SetColor` otherwise there's no link to go to :)

Reply:
ok, fixed that as well :-)